src/managers/PromptsManager.ts

```diff
@@ -124,6 +125,11 @@ export class PromptsManager {
       return;
     }
 
+    const { requiresUserInteraction } = EnvironmentInfoHelper.getEnvironmentInfo();
+    if (requiresUserInteraction && type==="native") {
```

Comment:
nit, `type==="native"` should be `type === DelayedPromptType.Native`

Reply:
true